cmd/config/set.go

```diff
@@ -0,0 +1,55 @@
+package config
+
+import (
+	"github.com/screwdriver-cd/sd-local/config"
+	"github.com/sirupsen/logrus"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	configNew = config.New
+)
+
+func newConfigSetCmd() *cobra.Command {
+	configSetCmd := &cobra.Command{
+		Use:   "set [key] [value]",
+		Short: "Set the config of sd-local",
+		Long: `Set the config of sd-local.
+Can set the below settings:
+* Screwdriver.cd API URL as "api-url"
+* Screwdriver.cd Store URL as "store-url"
+* Screwdriver.cd Token as "token"
+* Screwdriver.cd launcher version as "launcher-version"
+* Screwdriver.cd launcher image as "launcher-image"`,
+		Args: cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
If no error is returned, I think it is better to use `Run` instead of `RunE`.

Reply:
surely.
I fixed it.